besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1388,6 +1395,28 @@ private void validateDnsOptionsParams() {
     }
   }
 
+  private void validateGoQuorumOptions() {
+    if (isGoQuorumCompatibilityMode) {
+      final GenesisConfigOptions genesisConfigOptions = readGenesisConfigOptions();
+
+      if (!genesisConfigOptions.isQuorum()) {
+        throw new IllegalStateException(
+            "GoQuorum compatibility mode is enabled but genesis file 'isQuorum' flag isn't set to true. Are you using the correct genesis file?");
```

Comment:
"Are you using the correct genesis file" doesn't quite vibe with IllegalStateException. What about 
            "GoQuorum compatibility mode (enabled) can only be used if genesis file has 'isQuorum' flag set to true.");